### PR TITLE
button: give learn-more button consistent look globally, add button t…

### PR DESCRIPTION
**Pull request by waehlisch**
This PR updates the CSS to have a global look for the "Learn More" button. It also replaces the "Learn More" text with a button in the use case cards.

### CODE REVIEW

File: _includes/use-cases-carousel.html [hunk 1]
```diff
@@ -27,7 +27,9 @@ <h2>RIOT in the wild</h2>
                         <h3>{{ use_case.user }}</h3>
                         <h4>{{ use_case.user_position }}, {{ use_case.company }}</h4>
                         {% if use_case.project %}
-                          <a href="{{ use_case.url }}">Learn more</a>
+                        <div class="col-lg-12 mt-4 text-center">
+                          <a href="{{ use_case.url }}" class="btn-learn-more btn-primary btn-lg", role="button">Learn more!</a>
```
Comment (leandrolanzieri) [inline _includes/use-cases-carousel.html:31]: I think the `btn-primary` class is not adding additional properties here, and can be dropped

Comment (waehlisch) [reply _includes/use-cases-carousel.html:31]: i thought that this class defines color.

Comment (leandrolanzieri) [reply _includes/use-cases-carousel.html:31]: But now you added it to `btn-learn-more`

https://github.com/RIOT-OS/riot-os.org/pull/46/files#diff-605bda13bb3fd03734e5ec93bde9168b2b9d7959f942eb9e6b22c1df4365f8aeR78-R82

Comment (leandrolanzieri) [reply _includes/use-cases-carousel.html:31]: We could also keep the rest of properties in `btn-learn-more` and handle the colors with a separate class, if there are use cases for that.

Comment (waehlisch) [reply _includes/use-cases-carousel.html]: ok, i removed btn-primary.

**Review by leandrolanzieri — APPROVED** [review]
ACK!

**Pull request merged**